CHANGELOG.md

```diff
@@ -34,6 +34,10 @@
 
     *Manuel Puyol*
 
+* Fix `AvatarComponent` to apply classes to the link wrapper if present.
```

Comment:
mind moving this up? we tend to group breaking changes in the bottom

Reply:
👍